<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CilpsDatasetToolkit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="clipsListWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>581</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QVideoWidget" name="playerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>750</width>
      <height>422</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="clipsPathLineEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>520</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="c1PathLineEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>560</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="c2PathLineEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>600</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="c3PathLineEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>640</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="c4PathLineEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>680</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="clipsLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Path to clips</string>
    </property>
   </widget>
   <widget class="QLabel" name="c1Label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>560</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Category 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="c2Label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>600</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Category 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="c3Label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>640</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Category 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="c4Label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>680</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Category 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clipsPathButton">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>520</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c1PathButton">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>560</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c2PathButton">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>600</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c3PathButton">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>640</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c4PathButton">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>680</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1420</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PySide6.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
